src/App.vue

```diff
@@ -16,7 +16,7 @@
           </v-container>
         </v-main>
         <v-footer color="brown darken-4" app dark>
-          <span>2020</span>
+          <span class="white--text">&copy; {{ new Date().getFullYear() }}</span>
```

Comment:
Re-introduced the local date issue here.

Reply:
Update:

Here, is I change my date system settings to, say, 2019, the site will show copyright 2019. But that is false, as your site wasn't even there in 2019. You spread a false information then, and false copyright information is a serious legal issue.